interfaces.go

```diff
@@ -547,3 +565,19 @@ type EventHint struct {
 	Request            *http.Request
 	Response           *http.Response
 }
+
+// Log contains information about a logging event sent to Sentry.
+type Log struct {
+	Timestamp  time.Time            `json:"timestamp,omitempty"`
+	TraceID    TraceID              `json:"trace_id,omitempty"`
+	Level      Level                `json:"level,omitempty"`
+	Severity   int                  `json:"severity_number"`
```

Comment:
```suggestion
	Severity   int                  `json:"severity_number,omitempty"`
```

`severity_number` is optional, it can be ignored.